lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -45,12 +45,10 @@ public static function getObjectFieldValue($object, $field)
         if (is_array($object)) {
             return $object[$field];
         }
-
```

Comment:
Ah my bad, I was debugging a bit in a particular test case, I will revert this on Monday.